src/components/Settings/People/People.component.js

```diff
@@ -95,126 +90,125 @@ const People = ({
         disableSubmit={!isLogged}
       >
         <Paper>
-          <List>
-            <ListItem>
-              <div className="Settings__UserIcon__Container">
-                <UserIcon />
-              </div>
-              <ListItemText primary={name} />
-              <ListItemSecondaryAction className="Settings--secondaryAction">
+          <Box px={3} py={4}>
+            {/* Icono y Logout */}
+            <Box
+              display="flex"
+              flexDirection="column"
+              alignItems="center"
+              mb={4}
+            >
+              <UserIcon />
+              <Typography variant="h6" sx={{ mt: 2, textAlign: 'center' }}>
+                {name}
+              </Typography>
+              <Box display="flex" justifyContent="center" width="100%">
                 <Button
-                  disabled={!isLogged}
                   variant="outlined"
                   color="primary"
                   onClick={logout}
                   component={Link}
                   to="/"
+                  disabled={!isLogged}
+                  sx={{ mt: 2 }}
                 >
                   <FormattedMessage {...messages.logout} />
                 </Button>
-              </ListItemSecondaryAction>
-            </ListItem>
-            <ListItem>
-              <ListItemText
-                primary={<FormattedMessage {...messages.name} />}
-                secondary={<FormattedMessage {...messages.nameSecondary} />}
-              />
-              <ListItemSecondaryAction className="Settings--secondaryAction">
+              </Box>
+            </Box>
+
+            {/* Nombre */}
+            <Grid container spacing={2} alignItems="center" mb={3}>
+              <Grid item xs={12} sm={6}>
+                <Typography variant="subtitle1">
+                  <FormattedMessage {...messages.name} />
+                </Typography>
+                <Typography variant="body2" color="text.secondary">
+                  <FormattedMessage {...messages.nameSecondary} />
+                </Typography>
+              </Grid>
+              <Grid item xs={12} sm={6}>
                 <TextField
+                  fullWidth
                   disabled={!isLogged}
-                  id="user-name"
                   label={<FormattedMessage {...messages.name} />}
```

Comment:
Why you remove the id?

Reply:
I think is not necessary to duplicate the label as it is in the side (desktop) and in the top (mobile)